### PR TITLE
Added feature for noted creation

### LINKED ISSUE #6: Create new note UI

### PR DESCRIPTION
**Pull request by vysakh-poulose**
Fixes #6

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Form/index.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  };
+
+  return (
+    <Formik
+      initialValues={INITIAL_FORM_VALUES}
+      validationSchema={VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ isSubmitting }) => (
+        <FormikForm noValidate className="w-full">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Form/index.jsx:34]: Why do we need the `noValidate` class here?

File: app/javascript/src/components/Dashboard/Notes/Form/index.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+              label={t("notes.form.submit")}
+              loading={isSubmitting}
+              style="primary"
+              type="submit"
+            />
+            <Button
+              label={t("notes.form.cancel")}
+              style="text"
+              onClick={onClose}
+            />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Form/index.jsx:80]: Why doesn't the Cancel button have a type `reset`?

**Review by yedhink — COMMENTED** [review]
@vysakh-poulose

File: app/javascript/src/components/Dashboard/Notes/Form/index.jsx (hunk 1 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Notes/Form/index.jsx:34]: This was done to disable [validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes) from the HTML `form` element.

With `noValidate`            |  Without `noValidate`
:-------------------------:|:-------------------------:
Screenshot 2023-05-29 at 2 15 03 PM  |  Screenshot 2023-05-29 at 2 19 14 PM

File: app/javascript/src/components/Dashboard/Notes/Form/index.jsx (hunk 2 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Notes/Form/index.jsx:80]: Fixed in [80492b0](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/80492b041c62f399f806155507e1ec1f4b6dc38d)

File: app/javascript/src/components/Dashboard/Notes/Form/index.jsx (hunk 1 above)
Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Form/index.jsx:34]: Got it. Thanks for clarifying.

**Pull request merged**